app/Http/Controllers/Api/V1/SearchController.php

```diff
@@ -111,7 +131,8 @@ public function advance(SearchRequest $searchRequest)
      *
      * Dashboard search for projects, playlists and activities irrespective of indexing status
      *
-     * @queryParam  query required Query to search. Example: test
+     * @queryParam organization_id required The Id of a organization Example: 1
+     * @queryParam  query Query to search. Example: test
```

Comment:
Add a space before 'organization_id'

Reply:
Added a space before "organization_id" in "SearchController.php".